networkit/cpp/io/GraphReader.h

```diff
@@ -14,12 +14,26 @@
 
 namespace NetworKit {
 
+/*
+* Enum for defining handling of graph formats with multiple same edges.
+*/
+
+
+
 /**
  * @ingroup io
  * Abstract base class for graph readers.
  */
 class GraphReader {
 public:
+
+	enum MultipleEdgesHandling
+	{
+		 DISCARD_EDGES, //Reads and selects the first edge which occurs and discards all following
+		 SUM_WEIGHTS_UP, //If an edge occurs again, the weight of it is added to the existing edge
+		 KEEP_SHORTEST_PATH //The edge with the lowest weight is kept
```

Comment:
Wouldn't be something like `KEEP_MINIMUM_WEIGHT` more intuitive?

Reply:
Agreed.

Reply:
Fixed in https://github.com/kit-parco/networkit/pull/169/commits/109b56668449c7a52d332d2c7b50168cbd298800